ohai.gemspec

```diff
@@ -25,7 +25,6 @@ Gem::Specification.new do |s|
   s.add_dependency "ipaddress"
   s.add_dependency "wmi-lite", "~> 1.0"
   s.add_dependency "ffi", "~> 1.9"
-  s.add_dependency "chef-config", "~> 12.4"
```

Comment:
We will. Once there is a release of `chef-config` with [this change](https://github.com/chef/chef/commit/116e5219c717614658ecf2d7bf7f6b3420a8f6fe) we can re-add and update the version dep.